iot_packet_verifier/src/verifier.rs

```diff
@@ -163,9 +163,17 @@ impl Debiter for Arc<Mutex<HashMap<PublicKeyBinary, u64>>> {
     }
 }
 
+// TODO: Move these to a separate module
```

Comment:
this todo still applicable ?

Reply:
Yeah, I should move all the config client stuff into its own module. But later